recipes/cap3/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: cap3
+  version: '10.2011'
+
+source:
+  fn: cap3.linux.x86_64.tar # [linux]
+  url: http://seq.cs.iastate.edu/CAP3/cap3.linux.x86_64.tar # [linux]  
```

Comment:
Hello, I tried, but only the executable files are available.

(n.b : I'm a beginner bioinformatician here, sorry if I'm not answering correctly)